### PR TITLE
Add new case.qstatus script

### LINKED ISSUE #1584: Add a "case.qstatus" command
"case.submit" provides a flexible wrapper around the various qsub, brun, llsubmit, etc. commands that various schedulers use.

To unify the CIME user experience, we should add a "case.qstatus" command that will execute the appropriate "qstat, llq, whatever-slurm-uses" command to show the status of the submitted jobs in the queue.

### PR DESCRIPTION
**Pull request by jgfouca**
To make it easier to find the jobs associated with the case in
the queue.

Test suite: code_checker, by-hand
Test baseline: 
Test namelist changes: 
Test status: bit for bit

Fixes #1584 

User interface changes?: Yes, new case.qstatus tool!

Update gh-pages html (Y/N)?: N

Code review: jedwards, jacob

### CODE REVIEW

File: config/cesm/machines/config_batch.xml [hunk 1]
```diff
@@ -139,7 +139,7 @@
   </batch_system>
 
   <batch_system type="slurm" >
-    <batch_query>squeue</batch_query>
+    <batch_query per_job_arg="-j">squeue</batch_query>
```
Comment (jedwards4b) [inline config/cesm/machines/config_batch.xml]: Has this only been tested on slurm batch systems?  What about the others?

Comment (jgfouca) [inline config/cesm/machines/config_batch.xml]: Yes, I tested on Sandia's slurm machine chama. The PBS machines are easier because they don't need the -j.

**Comment by jgfouca** [comment]
@rljacob , since you requested this feature, I wanted to give you a chance to review it before it was merged.

**Comment by rljacob** [comment]
Tried it on blues and it works.  Is this part of the help string right?
```
positional arguments:
  caseroot       Case directory to build (default:
                 /lcrc/group/earthscience/jacob/cimeESMCI2/scripts/Xnewtool)
```

**Comment by jgfouca** [comment]
@rljacob nice catch, that was a copy/paste error. I did a quickie-push (directly to master) to change the wording.

**Pull request merged**